CONTRIBUTING.md

```diff
@@ -4,16 +4,15 @@ Thank you for your interest in contributing!
 
 Please ensure your pull request adheres to the following guidelines:
 
-
-- The purpose of this list is to increase the breadth of knowledge for testers and to discover new tools to help you on the way.
-- When you issue a pull request, you agree that your contribution is in the public domain (guided by the [CC-BY-SA license](LICENSE)).
-- Search previous suggestions before making a new one, as yours may be a duplicate.
-- Make an individual pull request for each suggestion. This allows to better handle updates or comments on the proposed character.
-- New categories, or improvements to the existing categorization are welcome.
-- Keep descriptions short and simple, but descriptive.
-- End all descriptions with a full stop/period.
-- Check your spelling and grammar.
-- Make sure your text editor is set to remove trailing whitespace.
+* The purpose of this list is to increase the breadth of knowledge for testers and to discover new tools to help you on the way.
```

Comment:
This change from "-" to "*" is unnecessary and makes it hard to review the changes. Please revert this!